ExpressWash/ExpressWash/User+CoreDataProperties.swift

```diff
@@ -34,9 +34,8 @@ extension User {
     @NSManaged public var zip: String?
     @NSManaged public var token: String?
     @NSManaged public var jobs: NSOrderedSet?
-    @NSManaged public var cars: NSOrderedSet?
+    @NSManaged public var cars: Set<Car>?
```

Comment:
I'm not sure about this. I think this should be left as an NSOrderedSet and then cast to a Set in a computed property.

```suggestion
    @NSManaged public var cars: NSOrderedSet?
...    
    public var carsArray: [Cars] {
      let set = cars as? Set<Car> ?? []
      return set.sorted {
        $0.carId < $1.carId
      }
    }
```